tests/admin_views/tests.py

```diff
@@ -4157,7 +4208,52 @@ def test_user_fk_popup(self):
         }
         response = self.client.post('/test_admin/admin/auth/user/add/?_popup=1', data, follow=True)
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, 'dismissAddAnotherPopup')
+        self.assertContains(response, 'dismissAddRelatedObjectPopup')
+
+    def test_user_fk_change_popup(self):
+        """Quick user change in a FK popup shouldn't invoke view for further user customization"""
```

Comment:
I am not sure what this docstring is trying to describe. Could you try rephrasing?